### DISCUSSION TITLE
Error [TCP] dial 208.95.112.1:80: HTTP connect status: 403 Forbidden"

### DISCUSSION THREAD

**Discussion by Raffy142** [xjasonlyu/tun2socks#422] [Q&A]
Can someone help me with this problem?
time="2024-10-17T15:27:26Z" level=info msg="[DIALER] set fwmark: 0x22b"
time="2024-10-17T15:27:26Z" level=info msg="[STACK] tun://tun0 <-> http://216.146.20.71:30438"
time="2024-10-17T15:27:30Z" level=info msg="[TCP] 198.18.0.1:38774 <-> 172.67.139.50:443"
time="2024-10-17T15:27:31Z" level=warning msg="[TCP] dial 208.95.112.1:80: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:31Z" level=info msg="[TCP] 198.18.0.1:38778 <-> 172.67.139.50:443"
time="2024-10-17T15:27:32Z" level=info msg="[TCP] 198.18.0.1:38790 <-> 172.67.139.50:443"
time="2024-10-17T15:27:32Z" level=info msg="[TCP] 198.18.0.1:57084 <-> 188.114.96.0:443"
time="2024-10-17T15:27:32Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:33Z" level=info msg="[TCP] 198.18.0.1:58942 <-> 104.21.57.199:443"
time="2024-10-17T15:27:33Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:34Z" level=info msg="[TCP] 198.18.0.1:55528 <-> 188.114.97.0:443"
time="2024-10-17T15:27:34Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:34Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:35Z" level=warning msg="[TCP] dial 43.154.120.135:6000: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:35Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:36Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"
time="2024-10-17T15:27:36Z" level=warning msg="[TCP] dial 207.246.121.171:7070: HTTP connect status: 403 Forbidden"

**Comment by MyNanyh** [comment]
我也有一样的问题[TCP] 198.18.0.1:37088 <-> 142.251.33.74:443
